app/views/histories/10_downing_street.html.erb

```diff
@@ -1,9 +1,20 @@
 <header class="govuk-grid-row">
   <div class="govuk-grid-column-full">
+    <%= render "govuk_publishing_components/components/breadcrumbs", {
+      breadcrumbs: [
+        {
+          title: "Home",
+          url: "/",
+        },
+        {
+          title: "History of the UK Government",
+          url: "/government/history",
```

Comment:
I do wonder if there's a way to automate these for breadcrumbs so no matter where it appears on the site - via the gem -  it's pre-populated.